translations/it/pack/tcu/uad.json

```diff
@@ -1,29 +1,62 @@
+        "traits": "Oggetto. Miglioria."
+    },
+    {
+        "code": "05231",
+        "flavor": "...e il potere corrompe.",
+        "name": "La Conoscenza è Potere",
+        "text": "Veloce. Gioca solo durante il tuo turno.\nScegli 1 supporto [[Tomo]] o [[Incantesimo]] che controlli oppure rivela 1 supporto [[Tomo]] o [[Incantesimo]] dalla tua mano. Risolvi una capacità [action] o [free] su quel supporto, ignorando tutti i costi (compresi gli eventuali costi [action]). Poi, se quel supporto era nella tua mano, puoi scartarlo per pescare 1 carta.",
+        "traits": "Intuizione."
     },
     {
         "code": "05232",
-        "name": "Esoteric Atlas",
-        "text": "Uses (4 secrets).\n[action] Spend 1 secret and exhaust Esoteric Atlas: Choose a revealed location that is exactly 2 connections away from your location. Move to that location.",
-        "traits": "Item. Tome.",
-        "slot": "Hand"
+        "name": "Atlante Esoterico",
+        "text": "Utilizzo (4 segreti).\n[action] Spendi 1 segreto ed esaurisci l’Atlante Esoterico: Scegli 1 luogo aperto che sia esattamente a 2 collegamenti dal tuo luogo. Muoviti fino a quel luogo.",
+        "traits": "Oggetto. Tomo.",
+        "slot": "Mano"
+    },
+    {
+        "code": "05233",
+        "name": "Investimento",
+        "text": "Utilizzo (0 scorte). Limite di 10 scorte sull’Investimento.\n[free] Esaurisci l’Investimento: Colloca 1 scorta su di esso.\n[action] Esaurisci e scarta l’Investimento: Sposta tutte le scorte dall’Investimento alla tua riserva delle risorse (trasformandole in risorse).",
+        "traits": "Contatto."
     },
     {
         "code": "05234",
-        "name": "Decoy",
-        "text": "<b>Evade.</b> Automatically evade a non-[[Elite]] enemy at your location.\n[reaction] When you play Decoy, increase its cost by 2: Change \"a non-[[Elite]] enemy\" to \"up to 2 non-[[Elite]] enemies.\"\n[reaction] When you play Decoy, increase its cost by 2: Change \"at your location\" to \"at a location up to 2 connections away.\"",
-        "traits": "Favor. Service."
+        "name": "Diversivo",
+        "text": "<b>Eludi.</b> Eludi automaticamente 1 nemico non [[Elite]] nel tuo luogo.\n[reaction] Quando giochi il Diversivo, aumenta di 2 il suo costo: Cambia \"1 nemico non [[Elite]]\" in \"fino a 2 nemici non [[Elite]]\".\n[reaction] Quando giochi il Diversivo, aumenta di 2 il suo costo: Cambia \"nel tuo luogo\" in \"in un luogo entro un massimo di 2 collegamenti\".",
+        "traits": "Favore. Servizio."
+    },
+    {
+        "code": "05235"
+        "flavor": "Il suo autore era Ludvig Prinn, perito sul rogo dell’inquisizione a Bruxelles quando i processi per stregoneria erano al loro apice.\n–Robert Bloch, \"L’Orrore dalle Stelle\"",
```

Comment:
missing commas at the end of some lines, so the json is invalid

Reply:
Sorry!
Should be fixed now